drainer/translator/mysql.go

```diff
@@ -56,9 +56,9 @@ func genMysqlInsert(schema string, table *model.TableInfo, row []byte) (names []
 	return names, args, nil
 }
 
-func genMysqlUpdate(schema string, table *model.TableInfo, row []byte, isTblDroppingCol bool) (names []string, values []interface{}, oldValues []interface{}, err error) {
+func genMysqlUpdate(schema string, table *model.TableInfo, row []byte, canAppendDefaultValue bool) (names []string, values []interface{}, oldValues []interface{}, err error) {
 	columns := writableColumns(table)
-	updtDecoder := newUpdateDecoder(table, isTblDroppingCol)
+	updtDecoder := newUpdateDecoder(table, canAppendDefaultValue)
```

Comment:
The function sign of `newUpdateDecoder` is still `(table *model.TableInfo, isTblDroppingCol bool) updateDecoder`.